01_temperature_processing.R

```diff
@@ -44,7 +45,7 @@ library(plyr)
 
 #Set the data file path and file that you want to process
 src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'
-SubID <- 'XXXX' # Enter the submission ID from VolWQDB
+SubID <- '0020' # Enter the submission ID from VolWQDB
```

Comment:
This is the key from the VolWQdb.t_Submission that is generated when I enter the following fields

Submission Description
Submission Organization
Project
Submission File Name (the uploaded Excel file)
Submission Date (to R)
Original File Name (sent to DEQ by organization)
Original Date (Date above sent to DEQ)
Submission Status.

To automatically generate this the uploaded Excel book would need to contain this information…(submission file name is already required by the script)

Steve Hanson
ODEQ Laboratory
Ph: 503.693.5737

From: petertbryant [mailto:notifications@github.com]
Sent: Friday, August 05, 2016 8:39 AM
To: petertbryant/ContinuousDataReview
Cc: HANSON Steve; Author
Subject: Re: [petertbryant/ContinuousDataReview] Comments for grade changes (#3)


In 01_temperature_processing.R<https://github.com/petertbryant/ContinuousDataReview/pull/3#discussion_r73713009>:

> @@ -44,7 +45,7 @@ library(plyr)

>

>  #Set the data file path and file that you want to process

>  src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'

> -SubID <- 'XXXX' # Enter the submission ID from VolWQDB

> +SubID <- '0020' # Enter the submission ID from VolWQDB

Can you explain to me where this number comes from? I'm wondering if there is a way to automate the step of getting the submission ID from the VolWQDB. Particularly so that we don't accidentally end up with duplicate IDs.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/petertbryant/ContinuousDataReview/pull/3/files/5e0a4e5e7e6880fd411f9290baa0ef06d04cfeb8#r73713009>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATJGv61Kx73PQ8HT580689zWc_ihnfnSks5qc1kIgaJpZM4Jb301>.